### PR TITLE
Add SIMLR 

### PR DESCRIPTION
**Pull request by sainirmayi**
## Describe your changes
Add a new SIMLR method to the dimensionality reduction task.

## Issue ticket number and link
Closes #xxxx (Replace xxxx with the GitHub issue number)

## Checklist before requesting a review
- [x] I have performed a self-review of my code

- Check the correct box. Does this PR contain:
  - [ ] Breaking changes
  - [ ] New functionality
  - [ ] Major changes
  - [ ] Minor changes
  - [ ] Bug fixes

- [x] Proposed changes are described in the CHANGELOG.md

- [ ] CI Tests succeed and look good!

### CODE REVIEW

File: src/tasks/dimensionality_reduction/methods/simlr/script.R [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  normalize = FALSE, 
+  cores_ratio = 1
+)
+meta <- list(
+  functionality_name = "simlr"
+)
+## VIASH END
+
+cat("Reading input files\n")
+input <- anndata::read_h5ad(par[["input"]])
```
Comment (KaiWaldrant) [inline src/tasks/dimensionality_reduction/methods/simlr/script.R:21]: just for future reference in r the items in a list can also be called with `par$input`. Much easier to type.

**Review by KaiWaldrant — CHANGES_REQUESTED** [review]
small changes

**Review by KaiWaldrant — APPROVED** [review]
LGTM

**Review by rcannood — APPROVED** [review]
looks good to me! minor suggestions for the R script

**Comment by rcannood** [comment]
Thanks @sainirmayi !

**Pull request merged**